frontend/awx/resources/inventories/InventoryForm.tsx

```diff
@@ -168,20 +185,19 @@ function InventoryInputs() {
   return (
     <>
       <PageFormTextInput
-        name="inventory.name"
+        data-cy="inventory-name"
```

Comment:
This data-cy is overwritten by the data-cy defined in the [PageFormTextInput](https://github.com/ansible/ansible-ui/blob/main/framework/PageForm/Inputs/PageFormTextInput.tsx#L264) component. This seems to be why the E2E test is failing while looking the element up by "inventory-name".